BackofficeBundle/Resources/public/coffee/table/tableHeader/TableviewSearchHeader.coffee

```diff
@@ -0,0 +1,27 @@
+TableviewSearchHeader = OrchestraView.extend(
+
+  tagName: "tr"
+  className: "header-search-input"
+
+  initialize: (options) ->
+    @options = options
+    @render()
+    widgetChannel.trigger 'ready', @
+    return
+
+  render: ->
+    for type, column in @options.inputHeader
+        type = "empty" if not tableFieldViewConfigurator[type]?
+        viewClass = tableFieldViewConfigurator[type]
```

Comment:
With your way, the `TableviewSearchHeader` file should be loaded first in all cases, I would rather suggest to do like a loading like for the viewConfigurator : https://github.com/open-orchestra/open-orchestra-media-admin-bundle/blob/master/MediaAdminBundle/Resources/public/coffee/loadBaseConfigurationView.coffee

Set the configuration with Jquery loaded